VideoLocker/src/main/java/org/edx/mobile/third_party/iconify/IconButton.java

```diff
@@ -0,0 +1,64 @@
+package org.edx.mobile.third_party.iconify;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.widget.ImageButton;
+
+import org.edx.mobile.R;
+
+public class IconButton extends ImageButton {
+
+    private int color;
+
+    public IconButton(Context context) {
+        super(context);
+    }
+
+    public IconButton(Context context, AttributeSet attrs) {
+        this(context, attrs, 0);
+    }
+
+    public IconButton(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        final TypedArray a = context.obtainStyledAttributes(
+                attrs, R.styleable.IconView, defStyleAttr, 0);
```

Comment:
>You might want to define a separate attribute set for this widget to enable independent extensibility. Obviously the large iconName enum set would need to be defined at the root level, and referenced in these attribute sets.

I dont see a need for this, both `IconView` and `IconButton` are using font-awesome as their base.

>Also, it looks like the default background is not being used anywhere, so I guess you should define a custom iconViewStyle theme attribute, and set it to a style that extends @android:style/Widget.Holo.Light.ImageButton, and nullifies the background.

I've defined an `iconButtonStyle`